google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/standard/AppEngineTemplateGroupDescriptorFactory.java

```diff
@@ -32,12 +33,19 @@
 
   @NonNls
   public static final String APP_ENGINE_WEB_XML_TEMPLATE = "GctAppEngineWeb.xml";
+  @NonNls
+  public static final String APP_YAML_TEMPLATE = "app.yaml";
```

Comment:
Maybe not for this PR, but it would be nice to find a suitable central place to store this. I noticed we're referring to it in string literals in several different places.